pyfar/plot/utils.py

```diff
@@ -1,47 +1,123 @@
+        | str  | The name of a style or a path/URL to a style file. For a    |
+        |      | list of available style names, see `style.available`.       |
+        +------+-------------------------------------------------------------+
+        | dict | Dictionary with valid key/value pairs for                   |
+        |      | `matplotlib.rcParams`.                                      |
+        +------+-------------------------------------------------------------+
+        | Path | A path-like object which is a path to a style file.         |
+        +------+-------------------------------------------------------------+
+        | list | A list of style specifiers (str, Path or dict) applied from |
+        |      | first to last in the list.                                  |
+        +------+-------------------------------------------------------------+
+
+    after_reset : bool
+        If True, apply style after resetting settings to their defaults;
+        otherwise, apply style on top of the current settings.
+
+    Examples
+    --------
+    >>> import pyfar
+    >>> import matplotlib.pyplot as plt
+    >>>
+    >>> with pyfar.plot.context():
+    >>>     fig, ax = plt.subplot(2, 1)
+    >>>     pyfar.plot.time(Signal([0, 1, 0, -1], 44100))
+    """
+
+    # get pyfar plotstyle if desired
+    style = plotstyle(style)
+
+    # apply plot style
+    with mpl_style.context(style):
+        yield
+
+
+def use(style="light"):
+    """
+    Use plot style settings from a style specification.
+
+    The style name of 'default' is reserved for reverting back to
+    the default style settings. This is a wrapper for `matplotlib.style.use`
+    that supports the pyfar plot styles 'light' and 'dark'.
+
+    .. note::
+
+       This updates the `.rcParams` with the settings from the style.
+       `.rcParams` not defined in the style are kept.
+
+    Parameters
+    ----------
+    style : str, dict, Path or list
+        A style specification. Valid options are:
+
+        +------+-------------------------------------------------------------+
+        | str  | The name of a style or a path/URL to a style file. For a    |
+        |      | list of available style names, see `style.available`.       |
+        +------+-------------------------------------------------------------+
+        | dict | Dictionary with valid key/value pairs for                   |
+        |      | `matplotlib.rcParams`.                                      |
+        +------+-------------------------------------------------------------+
+        | Path | A path-like object which is a path to a style file.         |
+        +------+-------------------------------------------------------------+
+        | list | A list of style specifiers (str, Path or dict) applied from |
+        |      | first to last in the list.                                  |
+        +------+-------------------------------------------------------------+
+
```

Comment:
Example would be nice :)
```suggestion
    Examples
    --------
import pyfar
import matplotlib.pyplot as plt

pyfar.plot.utils.use()

fig, ax = plt.subplots(2, 1)
pyfar.plot.time(pyfar.Signal([0, 1, 0, -1], 44100), ax=ax[0])
```